modules/administration-guide/pages/configuring-che-storage-size.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: CONCEPT
+:description: Configuring the storage size in {prod-short}
+:keywords: administration guide, configuring, {prod-id-short}, storage size, PVC size, pvc
+:navtitle: Configuring the storage size in {prod-short}
+:page-aliases: installation-guide:configuring-che-storage-size.adoc
+
+[id="configuring-{prod-id-short}-workspace-pvc-size"]
+= Configuring the storage size in {prod-short}
+
+The persistent volume claim (PVC) size can be configured when using the `per-user` or `per-workspace` storage strategies. PVC sizes in the Che Cluster Custom Resource must be specified in the format of a https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/[{kubernetes} resource quantity]. For more details on the available storage strategies, see xref:configuring-che-storage-strategy.adoc[]
+
+.Default persistent volume claim sizes:
+
+* `per-user`: `10Gi`
+* `per-workspace`: `5Gi`
+
+.Procedure
```

Comment:
After this point, we need a numbered list.
In AsciiDoc, the numbers can also be added automatically, so you can just start each step as follows (dot, space, sentence):
`. This is a sentence for step 1.`

Reply:
As it currently stands, the pages on configuring the storage strategy and configuring the storage size only have a single step. Should I still be using the `.Procedure` heading?